docs/source/delta-utility.md

```diff
@@ -719,4 +722,4 @@ CREATE OR REPLACE TABLE <target_table_name> SHALLOW CLONE iceberg.`/path/to/data
 .. <PrestoAnd> replace:: Presto and Athena
 
 .. <Delta> replace:: Delta Lake
-.. <AS> replace:: Apache Spark
\ No newline at end of file
+.. <AS> replace:: Apache Spark
```

Comment:
Don't know why this is showing up. There is no diff here.

Reply:
This PR added a new line.